src/modules/notes/NotesOverview.js

```diff
@@ -17,19 +17,18 @@ const SpotNotesOverview = props => {
   return (
     <View>
       {savedNote ? <View style={noteStyles.notesOverviewContainer}>
-        <View style={{flex: 4}}>
-          <Text style={props.style}>{truncateText(savedNote, 50)}</Text>
-        </View>
-        <View style={{flex: 1, justifyContent: 'flex-start'}}>
-          <View style={noteStyles.editButton}>
-            <Button
-              title={'Edit'}
-              onPress={() => dispatch(setNotebookPageVisible(NOTEBOOK_PAGES.NOTE))}
-              style={noteStyles.editButton}
-            />
-          </View>
+        <View>
+          <Text style={props.style}>{truncateText(savedNote, 750)}</Text>
         </View>
+        <View style={{flex: 1, justifyContent: 'flex-start'}}/>
       </View> : <Text style={commonStyles.noValueText}>No Notes</Text>}
+      {savedNote && <View style={noteStyles.editButton}>
```

Comment:
put parentheses around the View so the opening tag `<View` lines up with the closing tag `</View>`